test/e2e/annotations_test.go

```diff
@@ -0,0 +1,69 @@
+package e2e_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions"
+	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+var _ = Describe("Deploy deployment", Label("deployment-annotations-ns"), func() {
+	var data model.TestDataProvider
+
+	BeforeEach(func() {
+		Eventually(kubecli.GetVersionOutput()).Should(Say(K8sVersion))
+	})
+	AfterEach(func() {
+		GinkgoWriter.Write([]byte("\n"))
+		GinkgoWriter.Write([]byte("===============================================\n"))
+		GinkgoWriter.Write([]byte("Operator namespace: " + data.Resources.Namespace + "\n"))
+		GinkgoWriter.Write([]byte("===============================================\n"))
+		if CurrentSpecReport().Failed() {
+			GinkgoWriter.Write([]byte("Test has been failed. Trying to save logs...\n"))
+			utils.SaveToFile(
+				fmt.Sprintf("output/%s/operatorDecribe.txt", data.Resources.Namespace),
+				[]byte(kubecli.DescribeOperatorPod(data.Resources.Namespace)),
+			)
+			utils.SaveToFile(
+				fmt.Sprintf("output/%s/operator-logs.txt", data.Resources.Namespace),
+				kubecli.GetManagerLogs(data.Resources.Namespace),
+			)
+			actions.SaveTestAppLogs(data.Resources)
+			actions.SaveK8sResources(
+				[]string{"deploy", "atlasdeployments", "atlasprojects"},
+				data.Resources.Namespace,
+			)
+			actions.AfterEachFinalCleanup([]model.TestDataProvider{data})
+		}
+	})
+
+	DescribeTable("Namespaced operators working only with its own namespace with different configuration",
+		func(test model.TestDataProvider) {
+			data = test
+			mainCycle(test)
+		},
+		Entry("Simple configuration with keep resource policy annotation on deployment", Label("ns-crd"),
```

Comment:
Please add "deployment-annotations-ns" label

Reply:
Shouldn't we use the upper scope annotation to run all tests as a group?
https://github.com/mongodb/mongodb-atlas-kubernetes/blob/593d4c3ad440e2d6c917bf2df700685904a284b1/test/e2e/annotations_test.go#L17

Reply:
Yeah. Just trying to figure out why your tests aren't launching.